README.md

```diff
@@ -1181,7 +1181,7 @@ The `nf-core subworkflows create` command will prompt you with the relevant ques
 
 <!-- RICH-CODEX
 working_dir: tmp
-before_command: git clone https://github.com/nf-core/modules.git && cd modules
+before_command: rm -rf ~/.config/nfcore/nf-core/modules && git clone https://github.com/nf-core/modules.git && cd modules
```

Comment:
```suggestion
before_command: cd modules
```
We don't have to clone the repo again, it should already be there from the modules commands before (instead of `cd` we could remove the whole `before_command` and have `working_dir: tmp/modules`